recipes/oatpp-libressl/all/test_package/oatpp-libressl-test.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main() {
+    std::cout << "OK" << std::endl;	
+    exit(0);
```

Comment:
Can you please write a small test here?
It should just call some function from the library (it should call into a library, not test a compile definition nor a static inline function)

Reply:
@madebr Hi,
Thank you for suggestions. This is my first package to conan-center so my thanks are much greater.
According to call function from the library in test. Yes, generally it shoul but shoud doesn't mean must :) 

> The **test_package** differs from the library unit or integration tests, which should be more comprehensive. These tests are “package” tests, and validate that the package is properly created, and that the package consumers will be able to link against it and reuse it.

I'm disabling build tests while creating conan-library `self._cmake.definitions["OATPP_BUILD_TESTS"] = False`
so there is no need.